mypy.ini

```diff
@@ -1,4 +1,16 @@
 [mypy]
 python_version = 3.8
-ignore_missing_imports = True
 plugins = numpy.typing.mypy_plugin
+files = body, common, docs, scripts, selfdrive, site_scons, system, tools
```

Comment:
does this work? docs say it's supposed to be actual file paths, but glob patters are supported

Reply:
Seems to work fine!
```
((openpilot) ) batman@z840-openpilot4:~/openpilot$ mypy .
Success: no issues found in 469 source files
((openpilot) ) batman@z840-openpilot4:~/openpilot$ mypy
Success: no issues found in 469 source files
```